app/views/users/edit.html.erb

```diff
@@ -28,7 +28,10 @@
   <% @schedules.each do |schedule| %>
     <tr>
       <td><span class="glyphicon glyphicon-calendar" aria-hidden="true"></span> <%= "#{schedule.title}" %></td>
-      <td><%= link_to edit_user_schedule_path(@guider, schedule), title: "Edit Schedule #{schedule.title}", class: 'btn btn-info' do %><span class="glyphicon glyphicon-pencil" aria-hidden="true"></span><span class="sr-only">Edit Schedule</span><% end %></td>
+      <td>
+        <%= link_to edit_user_schedule_path(@guider, schedule), title: "Edit Schedule #{schedule.title}", class: 'btn btn-info' do %><span class="glyphicon glyphicon-pencil" aria-hidden="true"></span><span class="sr-only">Edit Schedule</span><% end %>
+        <%= button_to 'Delete', user_schedule_path(@guider, schedule), method: :delete, class: 'btn btn-danger t-delete' %>
```

Comment:
If you can get this to follow the same convention as the edit

with `<span class="glyphicon glyphicon-remove" aria-hidden="true></span><span class="sr-only">Delete Schedule</span>`